style.css

```diff
@@ -293,6 +293,12 @@ input[type="submit"]:hover {
     margin-right: 15%;
     
 }
+
+.about_us_text{
+	margin-left: 15%;
+        margin-right: 15%;
+	text-align: justify;
+}
 /* end of tre's stylization */
 
 /* styling for tutor page schedule */
```

Comment:
please don't put your name in the comments. the comments are about the code. "end of tre styli...." should be changed to a description of the code you are styling...

also, please fix your indentation before you push your code.

Reply:
did you fix that yet @TKTre